hipercam/scripts/makeflat.py

```diff
@@ -349,6 +349,9 @@ def makeflat(args=None):
                 os.close(fd)
 
                 # a bit of progress info
+                nframe = mccd.head["NFRAME"]
```

Comment:
I would delete these lines - the user doesn't actually need this information, and it looks a bit messy, even though it might be useful during debugging

Reply:
Ah you're right, I should have removed them before committing.